packages/reserve-site/src/service/addresses.ts

```diff
@@ -0,0 +1,33 @@
+import getAirtable, { Record, TableNames } from 'src/service/airtable'
+import { Addresses } from 'src/service/Data'
+const IS_LIVE = 'live=1'
+
+interface Fields {
+  live?: boolean
+  CELO: string
+  'CELO in custody': string
```

Comment:
Wasn't aware you could do this. Would a shorter one-word name work here instead?

Reply:
these are how the fields are named in Air table. I then convert to more js friend fields before using 🙂